cmd/context.go

```diff
@@ -0,0 +1,239 @@
+package cmd
+
+import (
+	"bufio"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/CircleCI-Public/circleci-cli/api"
+	"github.com/CircleCI-Public/circleci-cli/client"
+	"github.com/olekukonko/tablewriter"
+	"github.com/pkg/errors"
+
+	"github.com/CircleCI-Public/circleci-cli/settings"
+	"github.com/spf13/cobra"
+)
+
+func newContextCommand(config *settings.Config) *cobra.Command {
+
+	var cl *client.Client
+
+	initClient := func(cmd *cobra.Command, args []string) {
+		cl = client.NewClient(config.Host, config.Endpoint, config.Token, config.Debug)
+	}
+
+	command := &cobra.Command{
+		Use:   "context",
```

Comment:
Worth marking this hidden until we’ve done a few iterations so we can release it quietly early?

Reply:
+1 for hiding this subcommand. Then this PR can be merged separately from context group CRUD